src/aks-preview/HISTORY.rst

```diff
@@ -12,6 +12,7 @@ To release a new version, please select a new version number (usually plus 1 to
 Pending
 +++++++
 * Deprecate the alias "-r" of parameter --source-resource-id in `az aks trustedaccess rolebinding create`
+* Add `az aks nodepool delete-machines` command.
```

Comment:
Please rebase from main, the history note here is outdated.

Reply:
Fixed